### ISSUE TITLE
DLsite/F95Zone Error

### ISSUE THREAD

**Issue by ThePharaohArt** [erri120/Playnite.Extensions#6]
**Describe the bug**
When i try to download the metadata from either DLSite or F95Zone I get the error shown below. 

**Screenshots**
![image](https://user-images.githubusercontent.com/32932497/88356981-a0081100-cd37-11ea-9c38-aa1c3761fadb.png)

**Comment by erri120** [comment]
Need a little more information. Can you link the games and your log file?

**Comment by Poolhopper** [comment]
post from f95 zone
"hmm that did fix it but many other games do not work still im not sure what the common problem is between them.
here is a list of a few I found

https://f95zone.to/threads/amity-park-v0-7-3-gzone.4262/
https://f95zone.to/threads/the-mating-season-v1-02-akabur.13777/
https://f95zone.to/threads/princess-trainer-gold-edition-akabur.760/
https://f95zone.to/threads/poke-abby-hd-oxopotion.1068/

log for amity park

```
2020-07-22 04:44:25.4886|DEBUG:Downloading data from https://attachments.f95zone.to/2019/01/227040_banner_5.04.png to C:\Users\J\AppData\Local\Temp\Playnite\8c77edb1-6d82-45fb-9ce6-b4b4151d6cf2.png.
2020-07-22 04:44:25.8716|DEBUG:Downloading data from https://attachments.f95zone.to/2019/01/227055_screenshot11.jpg to C:\Users\J\AppData\Local\Temp\Playnite\1643b753-d98c-4eb5-ba51-413b53d31f1e.jpg.
2020-07-22 04:44:27.0506|DEBUG:Copying file C:\Users\J\AppData\Local\Temp\Playnite\8c77edb1-6d82-45fb-9ce6-b4b4151d6cf2.png to D:\Downloads folde\Games\Playnite77\library\files\aa51963d-b9a6-47ed-b3db-83f7fa459b24\5e0cce53-b5d3-4fed-adf3-336aef45ba3e.png
2020-07-22 04:44:27.0506|DEBUG:Copying file C:\Users\J\AppData\Local\Temp\Playnite\1643b753-d98c-4eb5-ba51-413b53d31f1e.jpg to D:\Downloads folde\Games\Playnite77\library\files\aa51963d-b9a6-47ed-b3db-83f7fa459b24\ad321957-8a77-4a78-8230-520c79ca9ca0.jpg
2020-07-22 04:45:04.4168|WARN:Found no Cover Image node for Amity Park [v0.7.3]!
2020-07-22 04:45:04.4168|ERROR:Failed to download metadata, 00000000-0000-0000-0000-000000000000, f8e1bcdb-b9d6-41f7-a8ae-799c504cb770System.AggregateException: One or more errors occurred. ---> System.ArgumentNullException: Value cannot be null.
Parameter name: oldChild
at HtmlAgilityPack.HtmlNode.RemoveChild(HtmlNode oldChild) in C:\Repos\HtmlAgilityPack\HtmlAgilityPack.Shared\HtmlNode.cs:line 1655
at F95ZoneMetadata.F95ZoneGame.<LoadGame>d__34.MoveNext() in M:\Projects\Playnite.Extensions\F95ZoneMetadata\F95ZoneGame.cs:line 143
--- End of inner exception stack trace ---
at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
at System.Threading.Tasks.Task`1.get_Result()
at F95ZoneMetadata.F95ZoneMetadataProvider.GetAvailableFields() in M:\Projects\Playnite.Extensions\F95ZoneMetadata\F95ZoneMetadataProvider.cs:line 54
at F95ZoneMetadata.F95ZoneMetadataProvider.get_AvailableFields() in M:\Projects\Playnite.Extensions\F95ZoneMetadata\F95ZoneMetadataProvider.cs:line 19
at F95ZoneMetadata.F95ZoneMetadataProvider.GetName() in M:\Projects\Playnite.Extensions\F95ZoneMetadata\F95ZoneMetadataProvider.cs:line 88
at Playnite.DesktopApp.ViewModels.GameEditViewModel.<>c__DisplayClass391_0.<DownloadPluginData>b__0() in E:\Devel\Playnite\source\Playnite.DesktopApp\ViewModels\GameEditViewModel.cs:line 2858
---> (Inner Exception #0) System.ArgumentNullException: Value cannot be null.
Parameter name: oldChild
at HtmlAgilityPack.HtmlNode.RemoveChild(HtmlNode oldChild) in C:\Repos\HtmlAgilityPack\HtmlAgilityPack.Shared\HtmlNode.cs:line 1655
at F95ZoneMetadata.F95ZoneGame.<LoadGame>d__34.MoveNext() in M:\Projects\Playnite.Extensions\F95ZoneMetadata\F95ZoneGame.cs:line 143<---"
```

**Comment by ThePharaohArt** [comment]
> Need a little more information. Can you link the games and your log file?

Gave me that error with any game that had no metadata... if it did have metadata the dialog for downloading the metadata from those websites would just not pop-up.

**Comment by erri120** [comment]
Fixed in 4127e8b07c687232062f2d2e710a4e90aa9b8e08, will soon release an update.
